keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -56,7 +58,7 @@ class VerificationRunner(
           if (newStatus.complete) {
             log.debug("Verification {} completed with status {} for {}", verification, newStatus, environment.name)
             markAs(verification, newStatus)
-            eventPublisher.publishEvent(VerificationCompleted(this, verification, newStatus))
+            eventPublisher.publishEvent(VerificationCompleted(this, verification, newStatus, state.metadata))
```

Comment:
It looks like this fires whenever any verification completes, which means that if there are multiple verifications, and it passes the first one but fails the second, it will still be tagged.

Reply:
I...uh...forgot you could have multiple verifications. Good catch.

Reply:
Added a new tag with the verification id to support looking for several tags in the "find image from tags" stage.